argon2.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: argon2
+  version: '20190702'
+  epoch: 0
+  description: The password hash Argon2, winner of PHC
+  copyright:
+    - paths:
+        - '*'
+      attestation: TODO
+      license: Apache-2.0
+  dependencies:
+    runtime:
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    environment:
```

Comment:
This part isn’t needed.

Reply:
Can you ellaborate this please?
I'll ofc remove that part, but tbh I do not fully understand why.

Reply:
It's because existing packages are pulled from a storage bucket and mounted to the local filesystem during CI.  This means we don't need to configure the repository or keyring which is used to serve up the same packages from the mounted storage bucket.

So you can remove the `repository` and `keyring` sections above, e,g it can look like this https://github.com/wolfi-dev/os/blob/6370c077ce82c9e50ba52a304d783acbc832a67a/git.yaml#L19-L21

We'll clean the existing melange configs up so it's consistent.

Reply:
Thanks, everything is clear now.